tools/quake3/common/threads.c

```diff
@@ -26,6 +26,8 @@
 #include <pthread.h>
 #endif
 
+#include <stdint.h>
```

Comment:
If this is only needed for uintptr_t below, then maybe move this inclusion to line 435?

Reply:
It seems strange to me to use an `#include` at the middle of a file.

Reply:
Well, true, but this is already what's done here, and given that this file contains arch-specific conditional compilation, it is perhaps not entirely unsurprising.